icons/openshift.svg

```diff
@@ -0,0 +1 @@
29,22.36a43.85,43.85,0,0,1-31.2-13.11Z"/><path d="M557.39,146.74l10.4-2.29v27.46a21.77,21.77,0,0,1,15.91-6.35c12.17,0,20.7,8.43,20.7,20.7v33.28H594V188c0-8.32-5.1-13.42-13.11-13.42a15,15,0,0,0-13.1,6.86v38.07h-10.4Z"/><path d="M615.21,151.62a6.39,6.39,0,0,1,6.35-6.44,6.45,6.45,0,0,1,0,12.89A6.39,6.39,0,0,1,615.21,151.62Zm11.55,67.92h-10.4V166.6h10.4Z"/><path d="M646.93,166.6v-8c0-11,6.34-17.05,18-17.05a27.45,27.45,0,0,1,7.49.93v9.05a20.77,20.77,0,0,0-6.55-.94c-5.72,0-8.53,2.6-8.53,8.22v7.8h15.08v8.74H657.33v44.2h-10.4v-44.2H634.66V166.6Z"/><path d="M688.22,175.34H677V166.6h11.23V153.08l10.29-2.5v16h15.6v8.74h-15.6V204c0,5.41,2.19,7.39,7.8,7.39a20.57,20.57,0,0,0,7.6-1.25v8.73a34.32,34.32,0,0,1-9.88,1.56c-10.3,0-15.81-4.88-15.81-14Z"/><path class="cls-1" d="M129,85c12.51,0,30.61-2.58,30.61-17.46a14,14,0,0,0-.31-3.42l-7.45-32.36c-1.72-7.12-3.23-10.35-15.73-16.6C126.39,10.19,105.26,2,99,2c-5.82,0-7.55,7.54-14.45,7.54-6.68,0-11.64-5.6-17.89-5.6-6,0-9.91,4.09-12.93,12.5,0,0-8.41,23.72-9.49,27.16A6.43,6.43,0,0,0,44,45.54C44,54.76,80.33,85,129,85m32.55-11.42c1.73,8.19,1.73,9.05,1.73,10.13,0,14-15.74,21.77-36.43,21.77C80,105.47,39.08,78.1,39.08,60a18.45,18.45,0,0,1,1.51-7.33C23.77,53.52,2,56.54,2,75.72,2,107.2,76.59,146,135.65,146c45.28,0,56.7-20.48,56.7-36.65,0-12.72-11-27.16-30.83-35.78"/><path d="M161.52,73.57c1.73,8.19,1.73,9.05,1.73,10.13,0,14-15.74,21.77-36.43,21.77C80,105.47,39.08,78.1,39.08,60a18.45,18.45,0,0,1,1.51-7.33l3.66-9.06A6.43,6.43,0,0,0,44,45.54C44,54.76,80.33,85,129,85c12.51,0,30.61-2.58,30.61-17.46a14,14,0,0,0-.31-3.42Z"/><path d="M581.24,94.3c0,11.89,7.15,17.67,20.19,17.67a52.11,52.11,0,0,0,11.89-1.68V96.51a24.84,24.84,0,0,1-7.68,1.16c-5.37,0-7.36-1.68-7.36-6.73V69.8h15.56V55.6H598.28v-18l-17,3.68V55.6H570V69.8h11.25Zm-53,.32c0-3.68,3.69-5.47,9.26-5.47a43.12,43.12,0,0,1,10.1,1.26v7.15A21.51,21.51,0,0,1,537,100.19c-5.46,0-8.73-2.1-8.73-5.57m5.2,17.56c6,0,10.84-1.26,15.36-4.31v3.37h16.82V75.58c0-13.56-9.14-21-24.39-21-8.52,0-16.94,2-26,6.1l6.1,12.52c6.52-2.74,12-4.42,16.83-4.42,7,0,10.62,2.73,10.62,8.31v2.73a49.53,49.53,0,0,0-12.62-1.58c-14.31,0-22.93,6-22.93,16.73,0,9.78,7.78,17.24,20.19,17.24m-92.44-.94h18.09V82.42h30.29v28.82h18.09V37.62H489.43V65.91H459.14V37.62H441.05ZM372.12,83.37c0-8,6.31-14.1,14.62-14.1a17.22,17.22,0,0,1,11.78,4.32V93a16.36,16.36,0,0,1-11.78,4.42c-8.2,0-14.62-6.1-14.62-14.09m26.61,27.87h16.83V33.94l-17,3.68V58.55a28.3,28.3,0,0,0-14.2-3.68c-16.19,0-28.92,12.51-28.92,28.5A28.25,28.25,0,0,0,383.8,112a25.12,25.12,0,0,0,14.93-4.83Zm-77.19-42.7c5.36,0,9.88,3.47,11.67,8.83H310c1.68-5.57,5.89-8.83,11.57-8.83M292.83,83.47c0,16.2,13.25,28.82,30.28,28.82,9.36,0,16.2-2.53,23.25-8.42l-11.26-10c-2.63,2.74-6.52,4.21-11.14,4.21a14.39,14.39,0,0,1-13.68-8.83h39.65V85.05c0-17.67-11.88-30.39-28.08-30.39a28.57,28.57,0,0,0-29,28.81M263.49,53.08c6,0,9.36,3.78,9.36,8.31s-3.37,8.31-9.36,8.31H245.61V53.08Zm-36,58.16h18.09V84.42h13.77l13.89,26.82h20.19l-16.2-29.45a22.27,22.27,0,0,0,13.88-20.72c0-13.25-10.41-23.45-26-23.45H227.52Z"/></svg>
```

Comment:
This is a rather large logo to fit in a small circle. Plus its not clear to me what the redistribution rights are on that logo. Since contains a trademarked image (the red hat) we'd need some explicit information somewhere on the Red Hat site saying that we fine to redistribute this logo. This repository is under an MIT license.

Reply:
I have pinged the Red Hat brand team to find this out.